htmd/builder/ionize.py

```diff
@@ -157,11 +157,10 @@ def ionizePlace(mol, anion_resname, cation_resname, anion_name, cation_name, nan
     if (nanion + ncation) == 0:
         return newmol
 
-    segname = _getSegname(newmol, segname)
     nions = nanion + ncation
 
-    betabackup = newmol.beta
-    newmol.set('beta', sequenceID((newmol.resid, newmol.segid)))
+    betabackup = newmol.beta.copy()
+    newmol.set('beta', sequenceID((newmol.resid, newmol.insertion, newmol.segid)))
```

Comment:
There are other calls to sequenceID around the code. Is this the only relevant one we should be fixing?

Reply:
Eh admittedly here it's overkill as it would work even without it. I added it for consistency. But if you find more wrong uses we could review them in another PR